test/e2e/aws_test.go

```diff
@@ -229,7 +229,7 @@ func TestAWSDefaults(t *testing.T) {
 		}
 	}
 
-	registryDeployment, err := client.Deployments(testframework.ImageRegistryDeploymentNamespace).Get(testframework.ImageRegistryName, metav1.GetOptions{})
+	registryDeployment, err := client.Deployments(testframework.ImageRegistryDeploymentNamespace).Get(testframework.ImageRegistryDeploymentName, metav1.GetOptions{})
```

Comment:
the change is fine, but i don't see how it's going to fix anything? these constants have the same value right?

Reply:
Fixed